umm-spec-lib/src/cmr/umm_spec/test/umm_g/expected_util.clj

```diff
@@ -92,7 +112,7 @@
                                 {:value "E51569BF48DD0FD0640C6503A46D4753"
                                  :algorithm "MD5"})
                     :size-unit "MB"
-                    :size 0.023
+                    :size (+ (double (/ 23552 (* 1024 1024))) (double (/ 11 1024)))
```

Comment:
why all the math and not a constant?

Reply:
because it's easier to see that it is the summation of the two sizes in DataGranule files.

Reply:
yea, but where does 23552 come from?

Reply:
A comment could be helpful but, it was hardcoded before this change